ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -51,6 +51,11 @@
   private static final ImmutableSet<String> FIREFOX_ONLY_DOCTYPES = ImmutableSet.of("event", "main",
       "modules");
 
+  private static final Map<String, String> IGNORED_URIS = ImmutableMap.<String, String>builder()
+      .put("/submit/sslreports", "1585144") //
+      .put("/submit/sslreports/", "1585144") //
```

Comment:
See https://bugzilla.mozilla.org/show_bug.cgi?id=1667827

Currently `/submit/sslreports` is dropped and `/submit/sslreports/` isn't, but it's probably best to scrub both just in case.

Reply:
I wonder if URI prefixes would be the more generally interesting target rather than specific URIs, in which case this would reduce to just "/submit/sslreports". But it would be easy to refactor this in the future if we end up with additional cases where we want to scrub based on prefix, so this looks fine as-is.